Update README with links to other stuff (#6)
Imageomics/bioclip: README.md

[Paper](https://arxiv.org/abs/2311.18803) | [Model](https://huggingface.co/imageomics/bioclip) | [Data](https://huggingface.co/datasets/imageomics/TreeOfLife-10M) | [Demo](https://huggingface.co/spaces/imageomics/bioclip-demo)
---

BioCLIP is a CLIP model trained on a new 10M-image dataset of biological organisms with fine-grained taxonomic labels.
BioCLIP outperforms general domain baselines on a wide spread of biology-related tasks, including zero-shot and few-shot classification.

## Table of Contents

1. [Model](#model)
2. [Data](#data)
3. [Paper, website, and docs](#paper)
4. [Citation](#citation)

## Model

The BioCLIP model is a ViT-B/16 pre-trained with the CLIP objective.
Both the ViT and the (small) autoregressive text encoder are available to download on [Hugging Face](https://huggingface.co/imageomics/bioclip).

The only dependency is the [`open_clip`](https://github.com/mlfoundations/open_clip) package.
See the `examples/` directory for examples on how to use it.

> To try it out, use this [Command Line Interface](https://github.com/Imageomics/bioclip-cli) or the [demo](https://huggingface.co/spaces/imageomics/bioclip-demo) on Hugging Face.

Reviewer: ```suggestion
To try it out, use our package, [`pybioclip`](https://github.com/Imageomics/pybioclip), or the [demo](https://huggingface.co/spaces/imageomics/bioclip-demo) on Hugging Face.
```
